dir-list-webpack-plugin.js

```diff
@@ -0,0 +1,81 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import fs from "fs";
```

Comment:
Not saying we do it now, but ...

`[promisify-node](https://github.com/nodegit/promisify-node)` helps with these.  I'd only suggest looking more seriously at it if more node packages need promisification.

Reply:
Also, Node 8 has a built in [`util.promisify`](https://nodejs.org/api/util.html#util_util_promisify_original).